mlrun/db/base.py

```diff
@@ -663,16 +663,16 @@ def create_user_secrets(
     @abstractmethod
     def create_model_endpoint(
         self,
-        project: str,
-        endpoint_id: str,
-        model_endpoint: Union[mlrun.model_monitoring.ModelEndpoint, dict],
-    ):
+        model_endpoint: mlrun.common.schemas.ModelEndpoint,
+    ) -> mlrun.common.schemas.ModelEndpoint:
         pass
 
     @abstractmethod
     def delete_model_endpoint(
         self,
+        name: str,
         project: str,
+        function_name: str,
         endpoint_id: str,
     ):
```

Comment:
I think we discussed it but I do not remember now - why can't we delete a MEP with just project + uid?

Reply:
The name is part of the path now so we have to get it.
So in order to get the desired deletion we need to pass (name, project, uid) or (name , project, function_name)